app/configurator/components/preview-breakpoint.tsx

```diff
@@ -149,7 +157,14 @@ export const PreviewBreakpointToggleMenu = ({
 const useStyles = makeStyles<Theme>((theme) => ({
   container: {
     width: "100%",
+    overflowX: "auto",
+    // Show the scrollbar on top
+    transform: "rotateX(180deg)",
+  },
+  chartContainer: {
     margin: "0 auto",
+    // Show the scrollbar on top
+    transform: "rotateX(180deg)",
```

Comment:
I do not understand this trick, what is it about ?

Reply:
It makes the scrollbar appear on top, otherwise it could sometimes be hard to understand that the area is scrollable, as the scrollbar would be outside of viewport as the bottom 👀 It's only scoped to the preview mode though, as in published mode the chart adapts to the viewport width :)

![top scrollbar](https://github.com/user-attachments/assets/d238b0da-4b16-42b8-8577-aef310d0f1df)

Reply:
So with double flip, everything looks the same, but the container scrollbar's position is reversed 😄

Reply:
Not a blocker: OK thanks, I wonder if this is intuitive enough for the user, as they are not accustomed to this. First time I see this need, I wonder if this is not indicative of a layout problem.

Reply:
With a layout problem, you mean that we can't show the scrollbar at the bottom of the viewport, even if the content is taller? I could also do it, but then the the vertical scroll would only happen in that container, instead of the whole layouting area, as is the case now. But maybe it would be better anyway, to keep the open sidebar / preview buttons always visible 👍